fe/fe-core/src/main/java/com/starrocks/http/rest/TransactionLoadAction.java

```diff
@@ -70,24 +94,30 @@
 import static com.starrocks.http.HttpMetricRegistry.TXN_STREAM_LOAD_PREPARE_NUM;
 import static com.starrocks.http.HttpMetricRegistry.TXN_STREAM_LOAD_ROLLBACK_LATENCY_MS;
 import static com.starrocks.http.HttpMetricRegistry.TXN_STREAM_LOAD_ROLLBACK_NUM;
+import static com.starrocks.http.rest.TransactionLoadAction.Operation.TXN_BEGIN;
+import static com.starrocks.http.rest.TransactionLoadAction.Operation.TXN_COMMIT;
+import static com.starrocks.http.rest.TransactionLoadAction.Operation.TXN_LOAD;
+import static com.starrocks.http.rest.TransactionLoadAction.Operation.TXN_PREPARE;
+import static com.starrocks.http.rest.TransactionLoadAction.Operation.TXN_ROLLBACK;
 
 public class TransactionLoadAction extends RestBaseAction {
     private static final Logger LOG = LogManager.getLogger(TransactionLoadAction.class);
+
+    private static final long DEFAULT_TXN_TIMEOUT_MILLIS = 20000L;
+
     private static final String TXN_OP_KEY = "txn_op";
-    private static final String TXN_BEGIN = "begin";
-    private static final String TXN_LOAD = "load";
-    private static final String TXN_PREPARE = "prepare";
-    private static final String TXN_COMMIT = "commit";
-    private static final String TXN_ROLLBACK = "rollback";
     private static final String TIMEOUT_KEY = "timeout";
     private static final String CHANNEL_NUM_STR = "channel_num";
     private static final String CHANNEL_ID_STR = "channel_id";
+    private static final String SOURCE_TYPE = "source_type";
+
     private static TransactionLoadAction ac;
 
     // Map operation name to metrics
-    private final Map<String, OpMetrics> opMetricsMap = new HashMap<>();
+    private final Map<Operation, OpMetrics> opMetricsMap = new HashMap<>();
```

Comment:
The name Operation is too ambiguous. Is TransactionOperation a better choice?

Reply:
Good suggestion, and i have renamed it as `TransactionOperation `